examples/geospatial-python-urban-analysis-with-postgis/src/etl/transform/district.py

```
@@ -0,0 +1,31 @@
+import pandas as pd
+import geopandas as gpd
+from config import EPSG_TARGET
```

Ruff (0.8.2)

`config.EPSG_TARGET` imported but unused

Remove unused import: `config.EPSG_TARGET`

(F401)